packages/website/builder/pages/index/index.js

```diff
@@ -146,8 +146,8 @@ Take a look at a real project using Cerebral, [Codesandbox.io](https://www.codes
             </a>
           </div>
           <div className="index-release">
-            Released under the MIT License
-            Copyright © 2017 Cerebral Github Organisation
+            Released under the MIT License Copyright © 2017 Cerebral Github
```

Comment:
PRobably we need to wrap lines with `<p>` and make year dynamic.

Reply:
Fixed. Used <br /> as <p> gave to much space.